### DISCUSSION TITLE
addMotifAnnotations only support homo sapiens, mus musculus with "cisbp " motifSet?

### DISCUSSION THREAD

**Discussion by tzhu-bio** [GreenleafLab/ArchR#985] [Questions / Documentation]
I would like to use **cisbp** motifSet with other species instead of _homo sapiens_ and _mus musculus_ in **addMotifAnnotation**.

Please tell me if there is any solution.

**Comment by rcorces** [comment]
we do not provide these annotations though you can create your own custom motif annotations. Unfortunately, we do not have a guide on how to do that yet so you would need to figure it out based on looking at how the human and mouse were created.
We use the format described in the `chromVARmotifs` package - https://github.com/GreenleafLab/chromVARmotifs

You can visualize these using:
```
library(chromVARmotifs)
data("human_pwms_v2")
human_pwms_v2
```
